2/debian-10/rootfs/opt/bitnami/scripts/kafka/postunpack.sh

```diff
@@ -9,8 +9,12 @@
 # Load Kafka environment variables
 eval "$(kafka_env)"
 
+# Copy server.properties from configtmp to config.
+# Temporary solution until kafka tarball places server.properties into config.
+cp "$KAFKA_BASEDIR"/configtmp/server.properties "$KAFKA_CONF_FILE"
```

Comment:
If we can remove this directory, even better:

```suggestion
mv "$KAFKA_BASEDIR"/configtmp/* "$KAFKA_CONFDIR"
rmdir "$KAFKA_BASEDIR"/configtmp
```